evennia/server/portal/ssh.py

```diff
@@ -74,6 +73,8 @@
     _PRIVATE_KEY_FILE, _PUBLIC_KEY_FILE
 )
 
+_BASE_SESSION = class_from_module(settings.BASE_SESSION_CLASS)
```

Comment:
Better call this _BASE_SESSION_CLASS to match what we call modules like this elsewhere.